repository/meeting-general.js

```diff
@@ -49,21 +49,16 @@ const closeMeeting = async (agendaURI) => {
   const query = `
   PREFIX besluit: <http://data.vlaanderen.be/ns/besluit#>
   PREFIX besluitvorming: <http://data.vlaanderen.be/ns/besluitvorming#>
-  PREFIX mulit: <http://mu.semte.ch/vocabularies/typed-literals/>
-  PREFIX ext:  <http://mu.semte.ch/vocabularies/ext/>
 
   DELETE {
-    ?meetingURI besluitvorming:behandelt ?oldAgenda ;
-      ext:finaleZittingVersie ?oldStatus .
+    ?meetingURI besluitvorming:behandelt ?oldAgenda .
   }
   INSERT {
-    ?meetingURI besluitvorming:behandelt ${sparqlEscapeUri(agendaURI)} ;
-      ext:finaleZittingVersie "true"^^mulit:boolean .
+    ?meetingURI besluitvorming:behandelt ${sparqlEscapeUri(agendaURI)} .
```

Comment:
Can we ensure this query still works as expected with mu-auth/Virtuoso, even if the `DELETE` clause has no matches. Uptil now `ext:finaleZittingVersie` was always set (either `true` or `false`) so there was always something to delete.
Alternative is to split in 2 queries, first a `DELETE` and afterwards an `INSERT`.

Reply:
I can confirm that this works. When I manually try to execute a DELETE then INSERT query where the DELETE doesn't have any matches, virtuoso still performs the INSERT:

```
callret-0
--
Delete from <http://www.openlinksw.com/schemas/virtrdf#DefaultSparul11Target>, 0 quads -- nothing to do Insert into <http://mu.semte.ch/graphs/organizations/intern-overheid>, 1 (or less) quads -- done Insert into <http://mu.semte.ch/graphs/organizations/intern-regering>, 1 (or less) quads -- done Insert into <http://mu.semte.ch/graphs/organizations/minister>, 1 (or less) quads -- done Insert into <http://mu.semte.ch/graphs/organizations/kanselarij>, 1 (or less) quads -- done
 ```
 
 (tested with:
 
 ```
 PREFIX besluit: <http://data.vlaanderen.be/ns/besluit#>
 PREFIX besluitvorming: <http://data.vlaanderen.be/ns/besluitvorming#>
 PREFIX mulit: <http://mu.semte.ch/vocabularies/typed-literals/>
 PREFIX ext:  <http://mu.semte.ch/vocabularies/ext/>

DELETE {
  GRAPH ?g {
    ?meetingURI besluitvorming:behandelt ?oldAgenda .
  }
}
INSERT {
  GRAPH ?g {
    ?meetingURI besluitvorming:behandelt <http://themis.vlaanderen.be/id/agenda/6374FA87D9A98BD0A228857A> .
  }
} WHERE {
  GRAPH ?g {
    ?meetingURI a besluit:Vergaderactiviteit .
    <http://themis.vlaanderen.be/id/agenda/6374FA87D9A98BD0A228857A> besluitvorming:isAgendaVoor ?meetingURI .
    OPTIONAL { ?meetingURI besluitvorming:behandelt ?oldAgenda . }
  }
}
 ```
 )